README.md

```diff
@@ -5,13 +5,61 @@ This repository is used to build the base container for gridappsd.
 It also includes a few scripts one to create the releases and another to create a blazegraph container.
 
 ./scripts/release/create_release.sh - Requires github username and token.  
-Release process:
-  * List open pull requests
-  * Create the release/$VERSION from develop
-  * Test the releases/$VERSION
-  * From the releases/$VERSION branch, create the pull request to master
-  * Assign and close the pull requests
-  * After the pull request's are approved then create the Tagged Releases
+
+To create a release, the create_release.sh script is run multiple times, between each run there are manual testing and validation steps.
+
+1.  Clone the build repository 
```

Comment:
should make the point that these should be parallel not embedded within

e.g. 
/root/gridappsd-docker
/root/gridappsd-docker-build

not 
/root/gridappsd-docker/gridappsd-docker-build

Reply:
I will update to make that more clear.